README.md

```diff
@@ -8,45 +8,56 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+efficient design and adoption of the highest quality C++ standard libraries**
+through implementation experience, user feedback, and technical expertise.
 
 ### Community
 
-We have two principle audiences: Library Developers and the C++ community.  We want to allow Library Developers to have a clear path on the road to Standardization.  And we want to make it easy for the C++ community to use these libraries to ensure we have real world usage.
+We have two principle audiences: Library Developers and the C++ community.
+We want to allow Library Developers to have a clear path on the road to
+Standardization.  And we want to make it easy for the C++ community to use
+these libraries to ensure we have real world usage.
 
 ### Governance
 
 This project is organized by our [Governance structure](docs/governance.md).
 
 ### Participating and contributing
 
-We welcome all to participate in the Beman project through 
-[Discourse](https://discourse.boost.org/t/welcome-to-beman-project-development/3) discussions or GitHub contributions. Our discourse community is the best way to stay up to
-date with the most recent developments.
+We welcome all to participate in the Beman project through
+[Discourse](https://discourse.boost.org/t/welcome-to-beman-project-development/3)
+discussions or GitHub contributions. Our discourse community is the best way to
+stay up to date with the most recent developments.
 
-If you are looking for ways to contribute code, see the open issues of [beman-project repositories](https://github.com/orgs/beman-project/repositories). The [beman repository's issue list](https://github.com/beman-project/beman/issues), in particular, has implementation requests for various papers. Issues marked with `good first issue` are perfect for new
-contributors and usually have an assigned mentor. Don't hesitate to post on discourse with any questions.
+If you are looking for ways to contribute code, see the open issues of
+[beman-project repositories](https://github.com/orgs/beman-project/repositories).
+The [beman repository's issue list](https://github.com/beman-project/beman/issues),
+in particular, has implementation requests for various papers. Issues marked
+with `good first issue` are perfect for new contributors and usually have an
+assigned mentor. Don't hesitate to post on discourse with any questions.
 
 Please refer to our [code of conduct](/docs/CODE_OF_CONDUCT.md) and the
-[Beman standard](/docs/beman-standard.md) for further information about the community and
-development guidelines.
+[Beman standard](/docs/beman-standard.md) for further information about the
+community and development guidelines.
 
 ### FAQ
 
 Questions? Maybe they have already been answered in our [FAQ](docs/FAQ.md).
 
 ### About the Name
 
-The Beman project is named in memory of Beman Dawes - co-founder of [Boost](https://www.boost.org).
+The Beman project is named in memory of Beman Dawes - co-founder of
```

Comment:
```suggestion
The Beman project is named [in memory of Beman Dawes](https://www.bemanproject.org/2024/10/30/about-beman/) - co-founder of
```

Reply:
We should delegate this to another PR, let's  keep this PR strictly formatting...